Sources/RepoToolsCore/RepoActions.swift

```diff
@@ -380,13 +380,13 @@ public enum RepoActions {
 
         let supportBUILDFile = assetRoot.appendingPathComponent("support")
             .appendingPathExtension("BUILD")
```

Comment:
Is this dead code or used anywhere? If not let's delete

Reply:
It's not dead code, it is the source file which is symlinked into pod_support_buildable from what I can tell and it creates a file called "support.BUILD" which seems fine